src/main/java/com/android/tools/build/bundletool/io/ApkSerializerManager.java

```diff
@@ -37,17 +37,9 @@
 import com.android.tools.build.bundletool.commands.BuildApksCommand.ApkBuildMode;
 import com.android.tools.build.bundletool.device.ApkMatcher;
 import com.android.tools.build.bundletool.io.ApkSetBuilderFactory.ApkSetBuilder;
-import com.android.tools.build.bundletool.model.ApkListener;
-import com.android.tools.build.bundletool.model.ApkModifier;
+import com.android.tools.build.bundletool.model.*;
```

Comment:
We're using Google java style guide which discourages the use of wildcard imports: 
https://google.github.io/styleguide/javaguide.html#s3.3.1-wildcard-imports